src/hooks/programPackage.ts

```diff
@@ -265,48 +265,33 @@ export const useDeliverProgramCollection = () => {
   return deliverPrograms
 }
 
-export const useGetProgramPackageCollection = (appId: string) => {
-  const { loading, error, data, refetch } = useQuery<hasura.GET_PROGRAM_PACKAGES, hasura.GET_PROGRAM_PACKAGESVariables>(
+export const useGetProgramPackageCollection = () => {
+  const { loading, error, data, refetch } = useQuery<hasura.GET_PROGRAM_PACKAGES>(
     gql`
-      query GET_PROGRAM_PACKAGES($appId: String!) {
-        program_package(where: { app: { id: { _eq: $appId } } }) {
+      query GET_PROGRAM_PACKAGES {
+        program_package {
           id
```

Comment:
`useGetProgramPackageCollection` is not a suitable naming for data hook.
You can deprecate it.
is it possible to combine `useProgramPackageCollection`data hook ?